code/game/objects/items.dm

```diff
@@ -149,18 +149,17 @@
 /obj/item/ex_act(severity, explosion_direction)
 	switch(severity)
 		if(0 to EXPLOSION_THRESHOLD_LOW)
+			explosion_throw(severity, explosion_direction)
```

Comment:
since this is under every item of the switch, you can just put this above the switch so it always fires